src/super_gradients/examples/yolov5_transfer_learning_example/yolov5_transfer_learning_example.py

```diff
@@ -0,0 +1,87 @@
+import super_gradients
+from super_gradients.training import SgModel, MultiGPUMode
+from super_gradients.training.datasets.dataset_interfaces.dataset_interface import \
+    PascalVOCUnifiedDetectionDataSetInterface
+from super_gradients.training.models.detection_models.yolov5 import YoloV5PostPredictionCallback
+from super_gradients.training.utils.detection_utils import base_detection_collate_fn
+from super_gradients.training.metrics import DetectionMetrics
+from super_gradients.training.utils.detection_utils import Anchors
+
+super_gradients.init_trainer()
+
+distributed = super_gradients.is_distributed()
+
+dataset_params = {"batch_size": 48,
+                  "val_batch_size": 48,
+                  "train_image_size": 512,
+                  "val_image_size": 512,
+                  "val_collate_fn": base_detection_collate_fn,
+                  "train_collate_fn": base_detection_collate_fn,
+                  "train_sample_loading_method": "mosaic",
+                  "val_sample_loading_method": "default",
+                  "dataset_hyper_param": {
+                      "hsv_h": 0.0138,  # IMAGE HSV-Hue AUGMENTATION (fraction)
+                      "hsv_s": 0.664,  # IMAGE HSV-Saturation AUGMENTATION (fraction)
+                      "hsv_v": 0.464,  # IMAGE HSV-Value AUGMENTATION (fraction)
+                      "degrees": 0.373,  # IMAGE ROTATION (+/- deg)
+                      "translate": 0.245,  # IMAGE TRANSLATION (+/- fraction)
+                      "scale": 0.898,  # IMAGE SCALE (+/- gain)
+                      "shear": 0.602,
+                      "mixup": 0.243
+                  }
+                  }
+
+
+model = SgModel("yolov5m_pascal_finetune_augment_fix",
+                multi_gpu=MultiGPUMode.OFF,
+                post_prediction_callback=YoloV5PostPredictionCallback())
+
+dataset_interface = PascalVOCUnifiedDetectionDataSetInterface(dataset_params=dataset_params, cache_labels=True, cache_images=True)
+model.connect_dataset_interface(dataset_interface, data_loader_num_workers=8)
+model.build_model("yolo_v5m", arch_params={"pretrained_weights": "coco"})
+
+post_prediction_callback = YoloV5PostPredictionCallback()
+
+network = model.net
+network = network.module if hasattr(network, 'module') else network
+num_levels = network._head._modules_list[-1].detection_layers_num
+train_image_size = dataset_params["train_image_size"]
+
+num_branches_norm = 3. / num_levels
+num_classes_norm = len(model.classes) / 80.
+image_size_norm = train_image_size / 640.
+
+training_params = {"max_epochs": 50,
```

Comment:
@shaydeci  - Should we have a recipe for this (not sure... just asking for your opinion...)

Reply:
mmm we talked about it when I started working on this, we said we'd start with a coded example. I can make a recipe out of it, but Im not sure what's better to demonstrate this concept for users....

Reply:
after discussing with @shaydeci, we agreed to create this code example for now. It allows debug and easier manipulations